documentation/book/proc-deploying-cluster-operator-kubernetes-to-watch-whole-cluster.adoc

```diff
@@ -0,0 +1,66 @@
+// Module included in the following assemblies:
+//
+// assembly-cluster-operator.adoc
+// assembly-operators-cluster-operator.adoc
+
+[id='deploying-cluster-operator-kubernetes-to-watch-whole-cluster-{context}']
+= Deploying the Cluster Operator to watch the whole cluster
+
+Cluster Operator can be installed to manage Kafka, Kafka Connect and Kafka Mirror Maker in the whole {ProductPlatformName} cluster across all {Namespaces}.
+
+.Prerequisites
+
+* Running {ProductPlatformName} cluster
+
+.Procedure
+
+. Edit the file `install/cluster-operator/050-Deployment-strimzi-cluster-operator.yaml` and set the value of the environment variable `STRIMZI_NAMESPACE` to `*`.
+For example:
++
+[source,yaml]
+----
+apiVersion: extensions/v1beta1
+kind: Deployment
+spec:
+  template:
+    spec:
+      serviceAccountName: strimzi-cluster-operator
+      containers:
+      - name: strimzi-cluster-operator
+        image: strimzi/cluster-operator:latest
+        imagePullPolicy: IfNotPresent
+        env:
+        - name: STRIMZI_NAMESPACE
+          value: "*"
```

Comment:
1. Are the quotes necessary? It would look prettier without them
2. I think we should add some ... to this to make is clearer it's a fragment of the `Deployment` yaml, not the whole thing.

Reply:
* I agree it would be nicer without the quotes ... but you get `error: error converting YAML to JSON: yaml: line 20: did not find expected alphabetic or numeric character`
* I will try to address the second point.